app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardActivity.kt

```diff
@@ -80,6 +81,7 @@ class DashboardActivity : DashboardElementActivity(), OnHomeItemClickListener, N
         super.attachBaseContext(LocaleHelper.onAttach(base))
     }
 
+    @SuppressLint("SetTextI18n")
```

Comment:
android studio give a suggestion on a better way to handle this instead of suppressing `
String literal in setText can not be translated. Use Android resources instead`